package.json

```diff
@@ -12,6 +12,7 @@
     "body-parser": "^1.18.2",
     "cors": "^2.8.4",
     "express": "^4.16.2",
+    "node-sass": "^4.12.0",
```

Comment:
it looks like you installed node-sass here, but this is the package.json for the server. to enable sass support you would have to add the dependency to the package.json for your react-app, and then alter the same file to UPGRADE your version of react-scripts, as the current version is showing `"react-scripts": "1.1.4"` which has does not include the built-in support for sass.